relayer/chain/ethereum/writer.go

```diff
@@ -80,7 +115,35 @@ func (wr *Writer) writeLoop(ctx context.Context) error {
 					return fmt.Errorf("Invalid message")
 				}
 
-				err := wr.Write(ctx, &concreteMsg)
+				err := wr.WriteChannel(ctx, &concreteMsg)
+				if err != nil {
+					wr.log.WithError(err).Error("Error submitting message to ethereum")
+				}
+			}
+		}
+	}
+}
+
+func (wr *Writer) writeBeefyLoop(ctx context.Context) error {
+	ticker := time.NewTicker(5 * time.Second)
```

Comment:
Can we put the seconds into a config variable?

Reply:
Removed this cron job entirely in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/cb1c68c0d84516ae5c1ee6344cafc44794a4ea35